webpack/server.js

```diff
@@ -3,9 +3,9 @@ import webpack from 'webpack'
 import config from './config'
 import { log } from '../src/lib'
 
-const webpackPort = 3000
+const webpackPort = process.env.WEBPACK_PORT
```

Comment:
maybe make this `const webpackPort = process.env.WEBPACK_PORT || 3000`
(and below: `const webpackHost = process.env.WEBPACK_HOST || 'localhost'` )